main.c

```diff
@@ -107,10 +107,11 @@ Address CreateInterface(const char* name, unsigned int slotCount) {
   //FIXME: Unsure about most terminology / inner workings here
   Address interfaceAddress = Allocate(1000); //FIXME: Size of object
   Address vtableAddress = Allocate(4 * slotCount);
+  char* slotNames = malloc(64 * slotCount);
   uint32_t* vtable = (uint32_t*)Memory(vtableAddress);
   for(unsigned int i = 0; i < slotCount; i++) {
     // Point addresses to themself
-    char* slotName = malloc(128);
+    char* slotName = &slotNames[i * 64];
```

Comment:
This seems like a stupid change; just calculate real length and stick to `malloc`